src/app/my-messages/skills-estimation-message-card/skills-estimation-message-card.component.html

```diff
@@ -10,7 +10,10 @@
     </div>
     <div fxLayout="row">
       <div class="messages-description-label">Message:</div>
-      <div [innerHTML]="buildSkillsText()" class="messages-message-text"></div>
+      <div class="messages-message-text">You need to estimate
+        the {{notification.skills.length > 1 ? 'skills' : 'skill'}} <strong>{{getAffectedSkills()}}</strong>. You can do
+        this on the <a routerLink="/my-skills">My skill profile</a> page
```

Comment:
I suggest to replace `on the My skill profile` with `in the My skill profile`. Ich denke es ist besser.